phplib/REST/Dashboard.php

```diff
@@ -39,16 +39,29 @@ public function GET(array $get) {
         $data['historical_alerts'] = $client->getAlertActivityCounts(self::RANGE);
 
         // Generate data for an Alert action histogram.
-        $sql = sprintf('
-            SELECT DATE(create_date, "unixepoch") as `date`, `action`, COUNT(*) as `count` FROM `%s` INNER JOIN (
-                SELECT DATE(create_date, "unixepoch"), `alert_id`, MAX(`create_date`) as `create_date`
-                FROM `%s` WHERE `site_id` = ? AND `archived` = 0 AND (
-                    (`action` = ? AND `a` = 1) OR
-                    (`action` = ? AND (`a` != 0 OR `b` != 0)) OR
-                    (`action` = ? AND `a` = 2)
-                ) AND `create_date` > ? AND DATE(create_date, "unixepoch") > DATE(?, "unixepoch") GROUP BY 1, 2
-            ) AS `tbl` USING(`alert_id`, `create_date`) GROUP BY 1, 2;
-        ', AlertLog::$TABLE, AlertLog::$TABLE);
+        if (DB::getType() == 'mysql') {
```

Comment:
Not a huge fan of this, but it works as an interim solution. #10 is intended to solve this.